RetailCoder.VBE/UI/RubberduckUI.fr.resx

```diff
@@ -1547,7 +1258,7 @@ Tous ceux qui nous ont donné une étoile ou un "like"
     <value>Supprimer...</value>
   </data>
   <data name="GeneralSettings_FolderDelimiterLabel" xml:space="preserve">
-    <value>Délimiteur de dossiers:</value>
+    <value>Délimiteur de dossier :</value>
```

Comment:
Stale key. These are notes to myself, really - you had no easy way of knowing this.